examples/MutableMap/MutableMapExamples.dfy

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT
+*******************************************************************************/
+
+// RUN: %dafny /compile:0 "%s"
+
+include "../../src/MutableMap/MutableMap.dfy"
+
+module MutableMapExamples {
+  import opened MutableMap
+
+  method Main() {
+    var m := new MutableMap<string,string>();
+    assert m.Keys() == {};
```

Comment:
I fully echo that there should be an executable example also, that tests all the calls and corner cases. Ideally it is the same method with both asserts and expects. Also should have example programs that test that preconditions are checked, along with testing that postconditions are strong enough to reason about.

Reply:
Unresolving this to track the fact that my suggestion above led to compilation errors, which we're going to figure out in case they are a real issue with using the library.

Reply:
Added `AssertAndExpect` and resolved compilation errors